config.py

```diff
@@ -1,72 +1,56 @@
-# coding: utf8
-"""
-config.py - Misc constants
-
-Copyright (c) 2018 The Fuel Rats Mischief,
-All rights reserved.
-
-Licensed under the BSD 3-Clause License.
-
-See LICENSE.md
-
-This module is built on top of the Pydle system.
-
-"""
+import json
 import logging
-
-
-class IRC:
-    """
-    IRC Configuration
-    """
-    ####
-    # Mecha's presence in IRC
-    presence = "Mechasqueak3-unknown[BOT]"
-    ####
-    # Server to connect to
-    server = "dev.localecho.net"
-    ####
-    # Port to connect on
-    port = "6667"
-    ####
-    # use TLS
-    tls = False
-    ####
-    # what channels to connect to
-    channels = ["#unkn0wndev"]
-
-    class Authentication:
-        """
-        Bots Authentication configuration
-
-        Currently this is just a skeleton and has no effect.
-        """
-        # TODO: Implement SASL authentication
-        username = ""
-        password = ""
-        SASL = True
-
-
-class Logging:
-    """
-    Log configurations
+import os
+from typing import Union
+
+CONFIGURATION: Union[None, dict] = None
+
+
+def setup_logging(root_logger: str, logfile: str):
```

Comment:
Does the logging setup belong here? I don't believe it has anything to do with the config, as such I feel it should not be in this module.

Reply:
Logging needs to be setup after the log file exists, I felt it was appropriate to call it after loading the configuration from disk